history-server.md

```diff
@@ -0,0 +1,71 @@
+# Spark History Server
+
+### Prerequisites
+
+Required software:
+* K8s cluster
+* [KUDO CLI Plugin](https://kudo.dev/docs/#install-kudo-cli) 0.7.3 or higher
+* [kubens](https://github.com/ahmetb/kubectx)
+
+### Installing Spark Operator with History Server Enabled
+
+```
+kubens spark-operator
+kubectl kudo install ./kudo-operator -p operatorImageName=mesosphere/kudo-spark-operator -p operatorVersion=latest -p enableHistoryServer=true
```

Comment:
let's put each param on a newline for better readability since the last param is arguably the most important. E.g.
```
kubectl kudo install ./kudo-operator \
  -p operatorImageName=mesosphere/kudo-spark-operator 
  -p operatorVersion=latest \
  -p enableHistoryServer=true